_posts/en/newsletters/2020-06-24-newsletter.md

```diff
@@ -0,0 +1,254 @@
+  the formula, its history, and even sample code from Bitcoin’s 0.1.5 release.
+
+- [Does Bitcoin really need timestamps?]({{bse}}96185)
+  Pieter Wuille explains why limiting the block rate without reference to a
+  clock time outside of the blockchain could make running full nodes more
+  expensive while also struggling to keep the stale block rate low and
+  preventing collusion attacks.
+
+- [In a fee overpayment attack, why can't compromised software provide fake previous transactions corresponding to fake inputs?]({{bse}}96309)
+  Regarding a [fee overpayment attack][news101 fee overpayment attack] on segwit
+  transactions with multiple inputs, justinmoon asks why the remedy of the
+  attack, requiring copies of previous transactions for the inputs, is not
+  vulnerable to malicious software providing fake previous transactions. Since
+  any provided previous transaction must have a hash that matches the spending input’s
+  previous transaction hash, such an attack is not feasible.
+
+## Releases and release candidates
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+- [LND 0.10.2-beta.rc2][lnd 0.10.2-beta] this release candidate for an
+  LND maintenance release is now available for testing.
+
+## Notable code and documentation changes
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo],
+[C-Lightning][c-lightning repo], [Eclair][eclair repo], [LND][lnd repo],
+[Rust-Lightning][rust-lightning repo], [libsecp256k1][libsecp256k1 repo],
+[Bitcoin Improvement Proposals (BIPs)][bips repo], and [Lightning
+BOLTs][bolts repo].*
+
+- [Bitcoin Core #19260][] disconnects peers that send the [BIP37][]
+  `filterclear` message if the local node isn't accepting bloom filters
+  (as advertised using the [BIP111][] `NODE_BLOOM` service flag).  It
+  was [previously proposed][Bitcoin Core #8709] that nodes starting in
+  Initial Blockchain Download (IBD) could register as non-relaying peers
+  to avoid receiving recent transactions while they were still
+  downloading large numbers of blocks.  When they finished syncing, they
+  could then transition to receiving relayed transactions by sending a
+  `filterclear` message.  However, it was more recently
+  [proposed][Bitcoin Core #19204] that this could be done instead using
+  the [BIP133][] `feefilter` message.  That eliminates any need for a
+  non-bloom node to support the `filterclear` message and so this PR
+  removes that feature.
+
+- [Bitcoin Core #19133][] adds a bitcoin-cli `-generate` parameter (note leading dash) to replace
+  the functionality of the bitcoind `generate` RPC method, which was removed
+  from Bitcoin Core in [Version 0.19.0.1][core version]. The new
+  implementation avoids creating unnecessary dependencies between the
+  wallet and other components. Providing an alias for the old RPC is useful for
```

Comment:
I suggested s/alias/client-side version/ because an alias for a no-longer existing RPC might be confusing and to highlight this is not a server-side RPC that is being restored. Could be wrong, but I suspect there is some confusion regarding the distinction (and why) between the RPC API and the CLI interface.

Reply:
Updated to `client-side alias` as a compromise.  I agree that there's confusion---I think the initial description here demonstrated that!